public/docs/ts/latest/guide/index.jade

```diff
@@ -9,121 +9,235 @@ figure
   This is a practical guide to Angular for experienced programmers who
   are building client applications in HTML and #{langName}.
 
+  Esta es una guía práctica de Angular para programadores con experiencia que
+  están construyendo aplicaciones cliente en HTML y #{langName}.
+
   We are on a journey together to understand how Angular works and, more importantly,
   how to make it work for us.  This overview begins the journey.
+
+  Estamos juntos en un viaje para entender cómo funciona Angular y, más
+  importante como hacerlo que nos ayude. Esta sección comienza el viaje.
+
   <br clear="all">
 // #enddocregion intro
-
 // #docregion how-to-read-1
 <a id="learning-path"></a>
 :marked
   # Organization
 
+  # Organización
+
   The documentation is divided into major thematic sections, each
   a collection of chapters devoted to that theme.
+
+  La documentación se divide en grandes secciones temáticas, con una
+  colección de capítulos dedicados a cada tema.
+
 // #enddocregion how-to-read-1
 // #docregion how-to-read-2
 - var top="vertical-align:top"
 table(width="100%")
   col(width="15%")
   col
   tr(style=top)
-    td <b>QuickStart</b>
+    td
+      p <b>QuickStart</b>
+
+      p <b>Inicio rápido</b>
+
     td
       :marked
         The foundation for every chapter and sample in this documentation.
+
+        La base para cada capítulo y ejemplo en esta documentación.
+
   tr(style=top)
-    td <b>Tutorial</b>
+    td
+      p <b>Tutorial</b>
+
     td
       :marked
         A step-by-step, immersive approach to learning Angular that
         introduces the major features of Angular in an application context.
+
+        Un enfoque paso a paso, de inmersión para aprender Angular que presenta
+        las características principales de Angular en el contexto de una aplicación.
+
```

Comment:
Hahaha era inmersivo!!! Note to self: no traducir y hablar con gente al mismo tiempo.